src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
```

Comment:
```suggestion
            <a href="#" class="navigation__link navigation__link--is-active">Apple</a>
```
Modifier should always go after the block/element name